.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,7 @@
   ⚠️    Changes in ops configuration etc. are required before deploying.
         [ Please add a link to the associated ops-issue or PR, such as in https://github.com/ITISFoundation/osparc-ops-environments or https://git.speag.com/oSparc/osparc-infra ]
   🗃️    Database table changed (relevant for devops).
+  👽️    Public API changes (meaning: dev features are moved to being exposed in production)
```

Comment:
@pcrespov @matusdrobuliak66 FYI I add here an icon to indicate changes in API server

Reply:
Haha, approved!

Reply:
probably add it to the title of the issue :+1:

Reply:
I added `include_in_schema=API_SERVER_DEV_FEATURES_ENABLED` to each of the endpoints, which means they are still dev features. I.e. they will be visible on the master deployment, but not staging and production. I will add the icon once this is removed and the features are exposed to the public (or the 👽️s)